File: Transport/OtherModels/tdbf/WaybillType.cs
using Kdn.Ext.Attributes;

namespace Transport.OtherModels.tdbf
{
    public class WaybillType
    {
        [IdProperty]
        public int waybillTypeId { get; set; }
        public string waybillTypeName { get; set; }
    }



}

File: Transport/JS/Transport/combo/WaybillType.js
T.combo.WaybillType = Ext.extend(Kdn.form.ComboBox, {
    initComponent: function() {
        Ext.applyIf(this, {
            displayField: 'WaybillTypeName',
            valueField: 'WaybillTypeId',
            store: Kdn.ModelFactory.getStore('WaybillType')
        });
        T.combo.WaybillType.superclass.initComponent.call(this);
    }
});

Ext.reg('combo.waybilltype', T.combo.WaybillType);